storage/innobase/xtrabackup/test/suites/xbcloud/storage_class.sh

```diff
@@ -1,6 +1,6 @@
 . inc/xbcloud_common.sh
 is_xbcloud_credentials_set
-
+is_minio_server && skip_test "Storage Class is not supported on MinIO servers"
 write_credentials
 
```

Comment:
at line 21 of subject we have a check
`if grep -q "minioadmin" $topdir/xbcloud.cnf ; then
  test_storage_class=false
fi`
are we passing some user other than minioadmin?

Reply:
That seems to be done as part of hiding password work - https://github.com/percona/percona-xtrabackup/blame/release-8.0.30-23/storage/innobase/xtrabackup/test/t/xbcloud.sh#L185

However, we better relay on server HTTP metadata do decide if we are running against a MinIO rather than a fixed user/password.

Reply:
agree.